### DISCUSSION TITLE
images are not loading in readme.md on mobile and android app

### DISCUSSION THREAD

**Discussion by ghost** [community/community#70557] [Mobile] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

I opened GitHub on my mobile and noticed that images are not showing in the `README.md` file. I also checked it on the GitHub Android app, and the images are not visible there either.

I tried removing any special characters from my file names, such as hyphens (-), but it did not resolve the issue. This problem is consistent across all repositories I have created.

For reference, I am accessing GitHub from `India` via `Jio mobile internet`.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @lisztomania23 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by RohitKumarKhatri** [comment]
```
![helloFile](../../attachments/01fig01.jpg)
![helloFile](/attachments/01fig01.jpg)
![helloFile](attachments/01fig01.jpg)

```
I tested all these three variations.
- Images are rendered in chrome
**- but not in the Github mobile app.**

Please check or suggest solution

**Comment by ghost** [comment, marked as answer]
tl;dr for [reference](https://www.reddit.com/r/developersIndia/comments/17wd709/jio_is_blocking_rawgithubusercontentcom_how_to/?utm_source=share&utm_medium=web2x&context=3)

It was due to Jio blocking raw.githubusercontent.com for no reason. 
Temporary Solution: Use Cloudflare's 1.1.1.1 DNS Resolver.

**Comment by Rahma765** [comment]
GitHub app English language change it to Maldives dhivehi language (ދިވެހި ބަސް)
